packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.js

```diff
@@ -19,6 +19,19 @@ const labels: $Shape<Labels> = {
 };
 
 describe("MultiSelect", () => {
+    beforeEach(() => {
+        window.scrollTo = jest.fn();
+
+        // We mock console.error() because React logs a bunch of errors pertaining
+        // to the use href="javascript:void(0);".
+        jest.spyOn(console, "error").mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+        window.scrollTo.mockClear();
+        jest.spyOn(console, "error").mockReset();
```

Comment:
Do you have to do this for the console error? I think this happens automatically.

Reply:
It happens automatically in the `webapp` repo, but we don't do it here. Probably worth copying that implementation in a separate PR.